src/components/AddBook.css

```diff
@@ -0,0 +1,71 @@
+form {
+  width: 87.6vw;
+  margin: 2.5rem 0.063rem 1.813rem 0;
+  border-top: solid 1px #e8e8e8;
+  margin-right: auto;
+  margin-left: auto;
+}
```

Comment:
### OPTIONAL


Zephin Design          |  Your design     
:-------------------------:|:-------------------------:
 ![Screenshot_2](https://user-images.githubusercontent.com/69638013/162593996-e4068bac-5030-452f-9f25-6ab08f0a6e7a.png) | ![Screenshot_3](https://user-images.githubusercontent.com/69638013/162593990-bbe2d57d-bfe4-4b8d-81b1-ee5ed6cf9708.png) 

- [ ] Kindly adjust the width of the form section to align with the width of the book container div, this will make your design look exactly like the Zephin design

Reply:
Thank for reviewing my pull request and I will fix it ASAP @prolajumokeoni  👍🏼